src/CcdImage.cc

```diff
@@ -195,8 +197,18 @@ CcdImage::CcdImage(lsst::afw::table::SortedCatalogT<lsst::afw::table::SourceReco
         ra = lsst::afw::geom::degToRad(RaStringToDeg(meta->get<std::string>("RA2000")));
         dec = lsst::afw::geom::degToRad(DecStringToDeg(meta->get<std::string>("DEC2000")));
     }
+    // TODO: Massive hack to get my test data to run.
+    // TODO: this all needs to go away once DM-5501 is dealt with.
```

Comment:
Yes and the whole if/else is a hack, but I agree that it's best to go with this code for now.